test/forge/integration/LiquidateIntegrationTest.sol

```diff
@@ -66,134 +66,130 @@ contract LiquidateIntegrationTest is BaseTest {
         morpho.liquidate(marketParams, BORROWER, amountSeized, 0, hex"");
     }
 
-    function testLiquidateSeizedInputNoBadDebt(
-        uint256 amountCollateral,
-        uint256 amountSupplied,
-        uint256 amountBorrowed,
-        uint256 amountSeized,
-        uint256 priceCollateral,
-        uint256 lltv
-    ) public {
-        _setLltv(_boundTestLltv(lltv));
-        (amountCollateral, amountBorrowed, priceCollateral) =
-            _boundUnhealthyPosition(amountCollateral, amountBorrowed, priceCollateral);
+    struct LiquidateTestParams {
+        uint256 amountCollateral;
+        uint256 amountSupplied;
+        uint256 amountBorrowed;
+        uint256 priceCollateral;
+        uint256 lltv;
+    }
 
-        vm.assume(amountCollateral > 1);
+    function testLiquidateSeizedInputNoBadDebt(LiquidateTestParams memory params, uint256 amountSeized) public {
```

Comment:
the name could be clearer. Are we testing a liquidation when the account isn't in a bad debt situation or when the liquidator doesn't realize the bad debt?

Reply:
```suggestion
    function testLiquidateSeizedInputNoBadDebtRealization(LiquidateTestParams memory params, uint256 amountSeized) public {
```